### ISSUE TITLE
Add examples to exported functions

### ISSUE THREAD

**Issue by jamesmbaazam** [epinowcast/epidist#157]
The key exported functions could benefit from having examples. I've noted `as_latent_individual()`, which tends to be called as the first step before `epidist()`. The latter, `epidist()`, also could benefit from the same suggestion.

**Comment by athowes** [comment]
I agree that we should add examples! Thanks for the suggestion, and for having a look at the package in these early stages more generally

**Comment by jamesmbaazam** [comment]
It's a pleasure. It's coming along nicely.

**Comment by athowes** [comment]
Note to self that this is a good one to work on soon.

**Comment by athowes** [comment]
@seabbs is this something we might want to consider as a part of public release?

**Comment by jamesmbaazam** [comment]
You'll need to do it if submitting to CRAN. It's a requirement.

**Comment by athowes** [comment]
Thanks @jamesmbaazam. The plan we have is an inital "public" release on GitHub / R Universe next week. (After a temporary "internal" release tomorrow.) So I think CRAN is a bit longer into the future. Nonetheless I think this is a useful feature (especially for new users to the package) and it'd be nice to have sooner rather than later.

**Comment by athowes** [comment]
PRs welcome if it's something you might feel up for tackling as part of your work!

**Comment by seabbs** [comment]
Mostly done in #513

**Comment by seabbs** [comment]
Im going to close this in favour of more targetted issues at remaining gaps
